windows/runner/Runner.rc

```diff
@@ -0,0 +1,121 @@
+// Microsoft Visual C++ generated resource script.
+//
```

Comment:
What's with all these windows related files?

We're not creating a windows application, it's polluting the codebase